src/hexkit/providers/akafka/provider/eventsub.py

```diff
@@ -44,6 +44,8 @@
 )
 
 ORIGINAL_TOPIC_FIELD = "original_topic"
+EXC_CLASS_FIELD = "exc_class"
+EXC_INFO_FIELD = "exc_info"
```

Comment:
In Python, [`exc_info`](https://www.oreilly.com/library/view/python-in-a/0596001886/re103.html) has a specific meaning as the 3-tuple of class, value and traceback.

Therefore it would be better to name the field differently, something like `EXC_MESSAGE_FIELD` = `"exc_message"` or `EXC_MSG_FIELD` = `"exc_msg"`.

Reply:
Thanks! I've changed it to "exc_msg". good suggestion